portfolio-edit/edit-project.php

```diff
@@ -3,15 +3,20 @@
 include './pullPortfolio.php';
 if (empty($_GET['id'])) {
     echo 'unexpected error no id set';
-}else {
+} else {
     $id = $_GET['id'];
-    $projects = pullPortfolio();
-    $currentProject;
-    foreach ($projects as $project) {
-        if ($project['id'] === $id) {
-            $currentProject = $project;
-        }
-    }
+
+
+    $db = new PDO('mysql:host=192.168.20.20;dbname=Project_2', 'root', '');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $sql = 'SELECT `id`, `name`, `url`, `comment` FROM `portfolio`WHERE `id` = :projectId;';
+    $query = $db->prepare($sql);
+    $query->execute([
+            ":projectId"=>$id
+    ]);
+    $result = $query ->fetchAll();
```

Comment:
if you use fetch instead of fetchAll you dont need the line below. and can just do:
```
$currentProject = $query->fetch();
```